tests/test_md_moves.py

```diff
@@ -0,0 +1,33 @@
+from typing import Tuple
+
+import numpy as np
+from scipy.stats import ks_2samp
+
+from timemachine.md.moves import MetropolisHastingsMove
+
+
+def test_rwmh():
+    """Test sampling from a 1-d normal distribution using Random Walk Metropolis-Hastings"""
+
+    np.random.seed(2023)
+    n_samples = 100_000
+    dx = 0.1
+
+    class RWMH1D(MetropolisHastingsMove[float]):
+        def propose_with_dlogq(self, x: float) -> Tuple[float, float]:
+            x_prop = x + np.random.uniform(-dx, dx)
+            log_q = -(x_prop ** 2) / 2
+            return x_prop, log_q
```

Comment:
should this be 
```suggestion
            log_q_0 = -(x ** 2) / 2
            log_q_prop = -(x_prop ** 2) / 2
            
            log_q_diff = log_q_prop - log_q_0
            
            return x_prop, log_q_diff
```
?

Reply:
Yes :facepalm: 

Fixed in https://github.com/proteneer/timemachine/pull/1134/commits/83205ba792c8f2ee7cbaa3a8557225a4794a827a

Reply:
A bit concerning that the test was still passing with this error. Will try to strengthen before merging.

Reply:
Strengthened the test in https://github.com/proteneer/timemachine/pull/1134/commits/dae47fbeb0f8052e54133588532bf878e125085f, adding a uniform sampling case and multiple seeds.